stdlib/experimental/experimental.flux

```diff
@@ -21,3 +21,27 @@ builtin to
 
 // An experimental version of join.
 builtin join
+
+// Aligns all tables to a common start time by using the same _time value for
+// the first record in each table incrementing all subsequent _time values by
+// using time elapsed between input records.
+// By default, it aligns to the _start value.
```

Comment:
So reading the implementation again I am wondering if we need this behavior. It adds ambiguity into the behavior of the function. For example which `_start` value should it use? If the are all the same then it doesn't matter, but if they differ we currently just pick the first record. What is special about the first record? Should the caller sort their data by `_start` first so they are confident the first record is meaningful? 

Thoughts? I see that the test case doesn't rely on this behavior rather instead it uses a specific `alignTo` parameter.

Reply:
It's meant to provide a reasonable default that falls inside of the queried time range. If there are more than one `_start` values, I don't know that it matters which one gets used, just that it doesn't default to the Unix epoch and doesn't force the user to define an `alignTo` time.

Reply:
From a UX perspective, if my queried time range goes back a day and the timestamps returned are from 1970, I might be a little confused.

Reply:
I guess I don't understand the use case of this function. It seems like it is basically a `timeShift` relative to a specific date. Can you maybe walk through the intended purpose of the function to help me understand?

The reason I am concerned about the `_start` as default is it requires that we run the query twice which is probably unexpected to the user. We run it once just to get the first record, then we run it again to map over the dates. This is a limitation of the way `tableFind` works.

Reply:
The primary use case is visualizing data in a way that aligns all series to the same starting point for direct comparisons over equal amounts of time. For example, here's some COVID-19 data:

![image](https://user-images.githubusercontent.com/181484/78192151-ccb8b580-7434-11ea-970a-e67bff996fe7.png)

Reply:
But to your point, in my actual visualization, I've replaced the time dimension with a day counter. Maybe this method of pulling the `_start` time from the data isn't necessary.

![image](https://user-images.githubusercontent.com/181484/78192361-3a64e180-7435-11ea-810e-f7d56ff721e5.png)

Reply:
@nathanielc I've been swayed and removed the `_start` default. It simplified the transformation significantly. Once you align time stamps, the significance of the timestamps lessens.

Reply:
Thanks for sharing the use case as well. The second I saw the graph it made sense to me the application of this function.